lib/src/strings.dart

```diff
@@ -0,0 +1,12 @@
+import 'objects.dart';
+
+/// True if the provided [String] value [v] is not null and is not blank
+/// (whitespaces)
+///
+/// `since 0.0.1`
+bool isNotBlank(String? v) => isNotNull(v) && v!.trim().isNotEmpty;
+
+/// True if the provided [String] value [v] is null or blank (whitespace only)
+///
+/// `since 0.0.1`
+bool isBlank(String? v) => isNull(v) || v!.trim().isEmpty;
```

Comment:
A function that checks every every character is whitespace (returning early as soon as the first non whitespace character is found) would be more efficient here.

Something like:
```suggestion
bool isBlank(String? v) => isNull(v) || v!.runes.every((c) => unicode.isWhiteSpace(c));
```
(but I don't know of a `unicode.isWhiteSpace` function)

One for Ron.
```suggestion
// TODO improver performance by checking if every character is whitespace
bool isBlank(String? v) => isNull(v) || v!.trim().isEmpty;
```

Reply:
Yeah, one doesn't exist that I know of